### ISSUE TITLE
redesign the registration login page  

### ISSUE THREAD

**Issue by viveksarkar0** [rahulsainlll/git-trace#77]
### Describe the documentation update.

it looks like facebook login not match well with the ui and aslo want to give some hover effect that match 

### Why is this needed?

cause it not match with the website aesthetic  i want to redesign it for you 

### Additional context

assgin me task me under the hacktoberfest and gssoc

**Comment by swarupn17** [comment]
Hey @viveksarkar0 I would like to work on this task. Please can you assign it to me.
Thanks !

**Comment by rahulsainlll** [comment]
I want retro design you guys have something in mind?

**Comment by viveksarkar0** [comment]
Yes the theme is grey so it should be gray type text alignment should be center just assigned me I will make it for you
